app-config.yaml

```diff
@@ -57,6 +57,18 @@ proxy:
     changeOrigin: true
     secure: true
 
+  '/sonarqube':
+    target: https://sonarcloud.io/api
+
+    allowedMethods: ['GET']
+    # uncomment if using a local instnace of sonarqube
+    # auth: "${SONARQUBE_TOKEN}:"
```

Comment:
The token should be enabled.

Reply:
again e2e test was failing if I remember correctly